pages/scripts/exporthardcopy.js

```diff
@@ -0,0 +1,271 @@
+var DOM = require('jsx-dom-factory');
+var _ = require('underscore');
+var $ = require('jquery');
+global.jQuery = $;
+var ElasticProgress = require('elastic-progress');
+
+var LighthouseJob = require('../lib/shared_job_code.js');
+var LighthouseUnit = require('../lib/shared_unit_code.js');
+var LighthouseJson = require('../lib/shared_json_code.js');
+
+// inject css c/o browserify-css
+require('../styles/exporthardcopy.css');
+
+var timeoverride = null;
+
+window.onerror = function(message, url, lineNumber) {
+  $('#loading')
+    .html('Error loading page<br>' + message + ' Line ' + lineNumber)
+    .show();
+  return true;
+};
+
+
+var selectedcolumns = [
+  'Identifier' ,  // Beacon Job ID
+  'JobReceived' , // Job Recieved Date/Time
+
+  'JobPriorityType.Name' , // Priority
+
+  'Address.PrettyAddress' ,         // Address
+  'Address.AdditionalAddressInfo' , // Additional Address Info
+
+  'PermissionToEnterPremises' , // Permission to Enter
+  'howToEnterPremises' ,        // How to Enter
+
+  'SituationOnScene' , // Situation On Scene
+  'Tags' ,             // Tags
+
+  'CallerFirstName' ,   // Caller - First Name
+  'CallerLastName' ,    // Caller - Last Name
+  'CallerPhoneNumber' , // Caller - Phone Number
+  'ContactFirstName' ,  // Job Contact - First Name
+  'ContactLastName' ,   // Job Contact - Last Name
+  'ContactPhoneNumber'  // Job Contact - Phone Number
+];
+
+
+var timeperiod;
+var unit = [];
+
+var params = getSearchParameters();
+
+
+/*
+var element = document.querySelector('.loadprogress');
+var mp = new ElasticProgress(element, {
+  buttonSize: 60,
+  fontFamily: 'Montserrat',
+  colorBg:    '#edadab',
+  colorFg:    '#d2322d',
+  onClose:function(){
+    $('#loading')
+    .hide();
+  }
+});
+*/
+RunForestRun();
+
+
+function getSearchParameters() {
+  var prmstr = window.location.search.substr(1);
+  return prmstr != null && prmstr != '' ? transformToAssocArray(prmstr) : {};
+}
+
+function transformToAssocArray(prmstr) {
+  var params = {};
+  var prmarr = prmstr.split('&');
+  for (var i = 0; i < prmarr.length; i++) {
+    var tmparr = prmarr[i].split('=');
+    params[tmparr[0]] = tmparr[1];
+  }
+  return params;
+}
+
+
+//Get times vars for the call
+function RunForestRun(mp) {
```

Comment:
i know you copy/pasted this but it would be great if it made more sense. Lower case first letter for functions (as opposed to classes) too.